leapp/libraries/stdlib/__init__.py

```diff
@@ -175,8 +177,13 @@ def run(args, split=False, callback_raw=_console_logging_handler, callback_lineb
     api.current_logger().debug('External command has started: {0}'.format(str(args)))
     _id = str(uuid.uuid4())
     result = None
+    _path = env.get('PATH', None) if env else None
     try:
         create_audit_entry('process-start', {'id': _id, 'parameters': args, 'env': env})
+        if not find_executable(args[0], _path):
+            #... just give something for audit - we count now to have always something inside...
+            result = {'exit_code': '1', 'stderr': 'File not found: {}'.format(args[0]), 'stdout': '', 'signal': 0, 'pid': 0}
```

Comment:
Hmm so you are setting result to this, but when is it going to be used?

Reply:
it's used in the final: statement. I was thinking originally about skipping it completely, but we already get used to having the result statement always available, so I decided to go with the way to keep it in this state. but it's discussable - cabal topic I (I will add it to the agenda in the morning). this is one of main reasons why postponed other tests and kept it in the draft state for now.